scripts/commitcheck.sh

```diff
@@ -125,6 +138,51 @@ function openzfs_port_commit()
     return $error
 }
 
+function is_coverity_fix()
+{
+    # subject starts with OpenZFS means it's an openzfs port
```

Comment:
Update the comment here to reflect this is a coverify not openzfs commit.

Reply:
Good catch, fixed.